### PR TITLE
chore: add sponsor detail

### LINKED ISSUE #53: Add Sponsors PDF in website
#### Reason/Context

Please try answering few of those questions
- Why we need this improvement?
We need to add PDF in the website.

- How will this change help?
Sponsors can download Sponsorship details and share to their team before making decision.

- What is the motivation? 
Discussed in https://github.com/asyncapi/community/discussions/74

#### Description
Please try answering few of those questions
- What changes have to be introduced?
A file to be uploaded in attachments folder.

- Will this be a breaking change?
No.

- How could it be implemented/designed?
A button will be added below sponsors heading and above `WANT TO BECOME A SPONSOR?` text

### PR DESCRIPTION
**Pull request by iamVP7**
**Description**

-  Add AsyncAPI-Hackathon-and-Conference-Prospectus.pdf in attachments folder
-  Add View Sponsorship Details in index.pug 

**Related issue(s)**
Fixes #53

### CODE REVIEW

File: sources/pug/pages/index.pug [hunk 1]
```diff
@@ -259,6 +259,11 @@ block page
   }).section
     #sponsors.container
       h3 Our sponsors
+      .row
+        .col-md-6.offset-md-3
+          a.button.button-primary.box-with-triangle-right.wow.fadeScale( href='/attachments/AsyncAPI-Hackathon-and-Conference-Prospectus.pdf' target="_blank" data-triangle=".button-overlay" )
```
Comment (derberg) [inline sources/pug/pages/index.pug:264]:
```suggestion
          a.button.button-primary.box-with-triangle-right.wow.fadeScale( href='https://conference.asyncapi.com/attachments/AsyncAPI-Hackathon-and-Conference-Prospectus.pdf' target="_blank" data-triangle=".button-overlay" )
```

this is the only way to solve it really, don't know any other solution than some hacks with JS. I think this "hardcode" hack is acceptable if you want it. I do not have a strong opinion here

Comment (iamVP7) [reply sources/pug/pages/index.pug:264]: I think it works..  tested in my local machine like below it worked.. 

```
a.button.button-primary.box-with-triangle-right.wow.fadeScale( href='http://localhost:8000/attachments/AsyncAPI-Hackathon-and-Conference-Prospectus.pdf' target="_blank" data-triangle=".button-overlay" )
```

**Review by derberg — APPROVED** [review]
Thanks a lot! I have to say that without your engagement we would probably end up not having the prospectus. Thanks 🙇🏼

**Comment by iamVP7** [comment]
All appreciation should goto @Barbanio & @fmvilas  for their effort in preparing the document. I just converted to PDF and  pushed it.

**Comment by Barbanio** [comment]
@iamVP7 For you too, it was teamwork 🚀

**Comment by fmvilas** [comment]
I just converted your document to a different design @iamVP7. It was definitely team work 👏

**Pull request merged**